.golangci.yml

```diff
@@ -223,25 +223,21 @@ linters:
     # - govet
     # - errcheck
     # - staticcheck
-    # - unused
+    - unused
     # - gosimple
-    # - structcheck
-    # - varcheck
     - ineffassign
-    # - deadcode
-    # - typecheck
+    - typecheck
     # Additional
     # - lll
     - godox
     # - gomnd
     # - goconst
     # - gocognit
-    # - maligned
```

Comment:
Why is this deleted? Are not we planning to enable it later?

Reply:
Maligned is also deprecated (replaced by govet)